src/wp-admin/css/colors/_mixins.scss

```diff
@@ -35,3 +35,12 @@
 		box-shadow: inset 0 2px 5px -3px darken( $button-color, 50% );
 	}
 }
+
+@mixin admin-scheme($color-primary) {
+	--wp-admin-theme-color: #{$color-primary};
+
+	// Darker shades.
+	--wp-admin-theme-color-darker-10: #{darken($color-primary, 5%)};
```

Comment:
Is the %age argument here intended to correspond to the `--wp-admin-theme-color-darker-##` value?

Reply:
We could pass the lightness/darkness values in as arguments, or handle that via a specific format - this was an initial idea inspire by @ryelle's post CSS themes: https://core.trac.wordpress.org/ticket/49999#comment:11

Reply:
> Is the %age argument here intended to correspond to the --wp-admin-theme-color-darker-## in the property name?

Agreed, that's weird, we can reconsider it but I just based this on the colors and variables used in Gutenberg.

Reply:
I like the idea of "lighter", "lightest"... instead of "lighter-10", "lighter-20".

Reply:
I think I'd prefer if we tweak the original styles to avoid variations as much as possible but that's a separate topic.

Reply:
Also - looking at some more examples in design systems, here are a couple that only use `-dark` `-xdark` and `-light` `-xlight` to abstract the property name from the darkness/lightness value:
* https://polaris.shopify.com/design/colors#section-color-palette
* https://bolt-design-system.com/pattern-lab/?p=viewall-visual-styles-color-palette

Reply:
I'd like to include @jasmussen in this discussion too as he helped a lot with the system in Gutenberg.

Reply:
Thanks for the ping, Riad, and thank you all for working on this.

I'm a little nervous about systematizing the lightness too much, because one of the big reasons for reducing the color palette was to actively discourage using too many colors. In that vein I'd honestly prefer to retire those darker values, if at all possible. But I know there are places where we can't do that, so in absence of that, we should try to avoid creating additional variables, and keep the lighter/darker steps to being as few as possible. Don't make 10 steps if we can do with 2. I sense that to also be the general vibe of the conversation here.